docs/en/observability/slo-overview.asciidoc

```diff
@@ -7,16 +7,23 @@
 
 beta::[]
 
-SLOs allow you to set clear, measurable targets for your service performance, based on factors like availability, response times, error rates, and other key metrics. 
+// tag::slo-license[]
+[IMPORTANT]
+====
+To create and manage SLOs, you need an {subscriptions}[appropriate license] and <<slo-privileges,SLO access>> must be configured.
+====
```

Comment:
@grabowskit confirmed that SLO access is limited to Platinum/Enterprise/Trial users so I added a note to the top of all pages that cover SLOs:

>To create and manage SLOs, you need an [appropriate license](https://www.elastic.co/subscriptions) and [SLO access](https://observability-docs_3331.docs-preview.app.elstc.co/guide/en/observability/master/slo-privileges.html) must be configured.

Reply:
I'm not opposed to this approach, but this would be a new pattern that, AFAIK, we don't replicate elsewhere in the docs. If we move forward with adding a banner to the top of every SLO page because the feature requires an elevated license, we should consider doing the same for other features in Obs that have the same requirement. For example:

<img width="818" alt="Screenshot 2023-11-08 at 3 35 50 PM" src="https://github.com/elastic/observability-docs/assets/5618806/50979dfc-5081-47a2-b5b9-270451253100">